crm_lead_type/models/crm_lead_type.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class CRMLeadType(models.Model):
+    _name = "crm.lead.type"
+    _description = "CRM Lead Type"
+
+    name = fields.Char("Name")
```

Comment:
Maybe we should make the field translatable like in sale_order_type addon ([here](https://github.com/OCA/sale-workflow/blob/13.0/sale_order_type/models/sale_order_type.py#L25))

Reply:
Agree

Reply:
Actually, we want to use here `sale.order.type`, rather than creating a new Model, so that this field can be passed on to the Quotation.